src/config.js

```diff
@@ -10,6 +10,7 @@ export default {
   messagesPerRequest: 20,
   scrollCallbackThrottle: 500,
   messageListThreshold: 500,
+  typingNotificationTimeout: 15 * 1000,
```

Comment:
There is no point in adding this to the config. We should hard code it.
Also, we likely want to remove more things from here later like messageListThreshold etc.